src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -1406,7 +1433,15 @@ public async Task ReinstateSubscriptionAsync(ISubscriber subscriber)
                     DefaultSource = defaultSourceId,
                     InvoiceSettings = new Stripe.CustomerInvoiceSettingsOptions
                     {
-                        DefaultPaymentMethod = defaultPaymentMethodId
+                        DefaultPaymentMethod = defaultPaymentMethodId,
+                        CustomFields = new List<Stripe.CustomerInvoiceSettingsCustomFieldOptions>
+                        {
+                            new Stripe.CustomerInvoiceSettingsCustomFieldOptions()
+                            {
+                                Name = "Organization",
```

Comment:
`UpdatePaymentMethodAsync` is used for both Users and Organizations, but here you're specifying `Organization`.

If the Name we give to Stripe can be anything, maybe we use the `SubscriberName` wording for that as well, so we don't have to distinguish between users and orgs.